test/e2e/suite/command/inspect.go

```diff
@@ -77,4 +77,58 @@ var _ = Describe("notation inspect", func() {
 				NoMatchErrKeyWords(HTTPSRequest)
 		})
 	})
+
+	It("sign with --force-referrers-tag set", func() {
+		Host(BaseOptions(), func(notation *utils.ExecOpts, artifact *Artifact, vhost *utils.VirtualHost) {
+			notation.Exec("sign", "--force-referrers-tag", artifact.ReferenceWithDigest()).
+				MatchKeyWords(SignSuccessfully)
+
+			notation.Exec("inspect", artifact.ReferenceWithDigest(), "-v").
+				MatchKeyWords(inspectSuccessfully...)
+		})
+	})
+
+	It("sign with --force-referrers-tag set to false", func() {
+		Host(BaseOptions(), func(notation *utils.ExecOpts, artifact *Artifact, vhost *utils.VirtualHost) {
+			notation.Exec("sign", "--force-referrers-tag=false", artifact.ReferenceWithDigest()).
+				MatchKeyWords(SignSuccessfully)
+
+			notation.Exec("inspect", artifact.ReferenceWithDigest(), "-v").
+				MatchKeyWords(inspectSuccessfully...)
+		})
+	})
+
+	It("sign with --allow-referrers-api set", func() {
+		Host(BaseOptionsWithExperimental(), func(notation *utils.ExecOpts, artifact *Artifact, vhost *utils.VirtualHost) {
+			notation.Exec("sign", "--allow-referrers-api", artifact.ReferenceWithDigest()).
+				MatchKeyWords(SignSuccessfully)
+
+			notation.Exec("inspect", artifact.ReferenceWithDigest(), "-v").
+				MatchKeyWords(inspectSuccessfully...)
+
+			notation.Exec("inspect", artifact.ReferenceWithDigest(), "--allow-referrers-api", "-v").
+				MatchErrKeyWords(
+					"Warning: This feature is experimental and may not be fully tested or completed and may be deprecated.",
```

Comment:
This line should not be emitted. What am I missing?

Reply:
This is because `--allow-referrers-api` flag is still kept under `experimental`. Its behavior is not changed for backwards compatible purpose.

Reply:
Why does this flag needs to be behind experimental flag, when we are going to keep implementation and are emitting warning?

Reply:
> Why does this flag needs to be behind experimental flag, when we are going to keep implementation and are emitting warning?

@priteshbandi `--allow-referrers-api` flag is already behind experimental as of today. I'm just keeping its behavior while hide the flag from new users only because we want to deprecate this experimental flag gracefully.

Reply:
@priteshbandi, does the comment from Patrick make sense to you? We want to deprecate the experimental flag gracefully and remove it in v1.3.0 release completely. In v1.2.0 release note, we will introduce the new flag `--force-referrers-tag` and mention the existing one `--allow-referrers-api` is deprecated and will be completely removed in v1.3.0. This is a more user-friendly approach and gives existing users a time window to replace the old flag with the new flag.